<?xml version="1.0" ?>
<sdf version="1.8">
  <model name="tractor_base" canonical_link="base">
    <static> true </static>
    <link name="base">
      <pose relative_to="__model__"> 0 0 0 0 0 0 </pose>
      <visual name="tractor_base">
        <laser_retro>1</laser_retro>
        <geometry>
         <mesh>
            <uri>models://meshes/simulation_assets/tractors/set_in_launch</uri>
          <scale>1 1 1</scale>
         </mesh>
       </geometry>
        <material>
          <ambient>0.1 0.1 0.1</ambient>
          <diffuse>0.1 0.1 0.1</diffuse>
          <specular>0.1 0.1 0.1</specular>
        </material>
      </visual>
    </link>
    <plugin
      filename="librandom_mover.so"
      name="lidar_sim::random_mover">
      <move_group>target</move_group>
      <range>7.0 31.0</range>
      <z_range>-0.1 0.1</z_range>
      <roll_range>-0.01 0.01</roll_range>
      <pitch_range>-0.01 0.01</pitch_range>
      <yaw_range>0 6.28</yaw_range>
      <appearance_p>1.0</appearance_p>
      <min_dist>5</min_dist>
    </plugin>
  </model>
</sdf>
